src/ts/browserutils.ts

```diff
@@ -25,6 +25,13 @@ export class BrowserUtils {
     return navigator && navigator.userAgent && /Android/.test(navigator.userAgent);
   }
 
+  static get isIOS(): boolean {
+    if (!this.windowExists()) {
+      return false;
+    }
+    return navigator && navigator.userAgent && (/iPad|iPhone|iPod/.test(navigator.userAgent) || (navigator.platform === 'MacIntel' && navigator.maxTouchPoints > 1));
```

Comment:
I would prefer to split this up to prevent confusions in the future:
- `isIOS`
- `isMacOS` / `isMacIntel`
- `isTouchSupported` (extract from `seekbar.ts`)

and then use it like `isIOS || (isMacOS && isTouchSupported)`